2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
+  [[ -n "$export_str" ]] && export $export_str
+
+  if [[ -z "${MEMORY_LIMIT_IN_BYTES:-}" ]]; then
+    echo "Warning: Can't detect memory limit"
+  fi
+  if [[ -z "${NUMBER_OF_CORES:-}" ]]; then
+    echo "Warning: Can't detect number of cores"
```

Comment:
we do but it might mean that something went wrong in the script. Unless there's a case where cgroups are not used. I don't think the warning would be harmful in that case, I assume it'll be a minority issue?